src/AppBundle/Entity/Report/Traits/ReportProfDeputyCostsTrait.php

```diff
@@ -401,4 +401,15 @@ public function generateActualSubmittedOtherCosts()
 
         return $submittedCosts;
     }
+
+    /**
+     * Has at least one other cost been submitted? Used to determine whether section is complete as question is last
+     * to be asked.
+     *
```

Comment:
Can we remove the comment please? It's a generic method that ought not really know the intentions of whatever calls it. That intent should be clear in the calling code, and you have made it clear by naming the calling method `breakdownCostsIncomplete`.

Reply:
Removed.